manager/pom.xml

```diff
@@ -63,15 +66,8 @@
       <scope>test</scope>
     </dependency>
     <dependency>
-      <groupId>org.testcontainers</groupId>
-      <artifactId>testcontainers</artifactId>
-      <version>1.16.0</version>
-      <scope>test</scope>
-    </dependency>
-    <dependency>
-      <groupId>org.testcontainers</groupId>
-      <artifactId>postgresql</artifactId>
-      <version>1.16.0</version>
+      <groupId>io.quarkus</groupId>
+      <artifactId>quarkus-jdbc-h2</artifactId>
```

Comment:
fyi I've removed test container since I've managed to run the tests without any mock for the persistence layer using h2, which is much faster than running a testcontainer

Reply:
How big is this difference? Because it could be slower but at least we prevent errors related to different SQL syntax and similar that we already faced in the past. Personally I would probably prefer to use _the real_ env

Reply:
Honestly I did not measure it, but **it is** faster because h2 is running in memory and testcontainer has to start a docker container (after having downloaded it in case), init the application and so on and so forth. I've created https://issues.redhat.com/browse/MGDOBR-26 so that we can move forward with this PR, is that ok for you @danielezonca ?